lnwire/accept_channel.go

```diff
@@ -156,7 +166,8 @@ func (a *AcceptChannel) Decode(r io.Reader, pver uint32) error {
 	if err != nil && err != io.EOF {
 		return err
 	}
-	return nil
+
```

Comment:
reminder that how to handle the above upfront shutdown must be decided on in the spec